clients/src/main/java/org/apache/kafka/clients/ClientDnsLookup.java

```diff
@@ -0,0 +1,38 @@
+package org.apache.kafka.clients;
```

Comment:
license is missing

Reply:
good catch, fixed, thank you

Reply:
Can you fix checkStyle issues and rebase the PR?
./gradlew :clients:checkstyleMain

Reply:
Couldn't find checkStyle issues : 
:clients:compileJava UP-TO-DATE
:clients:processResources NO-SOURCE
:clients:classes UP-TO-DATE
:clients:checkstyleMain UP-TO-DATE

BUILD SUCCESSFUL in 11s
2 actionable tasks: 2 up-to-date
10:58:05: External task execution finished 'checkstyleMain'.

Reply:
rebased by squashing all commits.